Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Pyhton image
+
+FROM python:2.7-slim
+
+# System requirements
+
+RUN apt-get -y update && apt-get -y install \
+    gcc \
+    python-setuptools \
+    python-dev \
+    python-mysqldb \
+    libxml2-dev \
+    libxslt1-dev \
+    lib32z1-dev \
+    libpq-dev \
+    libldap2-dev \
+    libsasl2-dev
```

Comment:
Maybe we should build this step into a common image, and use it for all of our Django-based docker applications?

We can have for example: `eeacms/python:2.7-slim` as a base image.